crates/wascc-host/src/messagebus/rpc_client.rs

```diff
@@ -151,19 +190,54 @@ impl Handler<LinkInbound> for RpcClient {
         }
     }
 }
-// Publish a link to the RPC bus
+// Publish a link definition to the RPC bus
 impl Handler<AdvertiseBinding> for RpcClient {
-    type Result = Result<()>;
+    type Result = ResponseActFuture<Self, Result<()>>;
 
     fn handle(&mut self, msg: AdvertiseBinding, _ctx: &mut Self::Context) -> Self::Result {
-        Ok(())
+        trace!("Publishing link definition on lattice");
+        let ld = LinkDefinition {
+            actor: msg.actor,
+            contract_id: msg.contract_id,
+            link_name: msg.binding_name,
+            provider_id: msg.provider_id,
+            values: msg.values,
+        };
+        let nc = self.nc.clone().unwrap();
+        let subject = links_subject(&self.ns_prefix);
+        let bytes = serialize(&ld).unwrap(); // we should never fail or own serialize
```

Comment:
should this be "our own serialize"

cmon it's not a PR review from me if I don't nitpick spelling somewhere right

Reply:
(╯°□°)╯︵ ┻━┻